tests/test_integration_scenarios.py

```
@@ -0,0 +1,512 @@
+                    MagicMock(
+                        id="12345",
+                        title="Test",
+                        reward=25.0,
+                        url="https://example.com",
+                        source="rss",
+                    )
+                ]
+            }
+        )
+
+        async def mock_accept(job):
```

Ruff (0.14.14)

Unused function argument: `job`

(ARG001)